R/el_cuml.R

```diff
@@ -222,3 +222,30 @@ get_partners <- function(dat, index_posit_ids, networks = NULL,
 
   return(partner_df)
 }
+
+#' @title Return the Cumulative Degree of a Set of Index Nodes
+#'
+#' @inheritParams get_partners
+#'
+#' @return
+#' A \code{data.frame} with 2 columns:
+#' \itemize{
+#'   \item \code{index}: the unique ID (see \code{get_unique_ids}) of the
+#'         indexes.
```

Comment:
So this takes as an argument the posit ID but returns the unique ID. Is that necessary/justified? Is there any case where we are getting the cumulative degree for nodes without a posit ID (i.e., nodes who are already departed from the network)?

Reply:
Good catch!
You are right in this case it does not really make sense as we only work with information on active nodes anyways.
I will fix

Reply:
Great, thanks!